src/TAlias.cpp

```diff
@@ -350,20 +343,13 @@ void TAlias::compileAll()
     if( ! compileScript() )
     {
         if( mudlet::debugMode ) {
-            TDebug( QColor(Qt::white), QColor(Qt::red) ) << "LUA ERROR: when compiling script of alias:"
+            TDebug( QColor(Qt::white), QColor(Qt::red) ) << "ERROR: Lua compile error. compiling script of alias:"
                                                          << mName
                                                          << "\n"
                                                          >> 0;
         }
         mOK_code = false;
     }
-    else
```

Comment:
Have taken this message back out to keep the behaviour like it was before (I think).